templates/results.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="description" content="Mini Project to test out technologies for the Isuroon project." />
+    <meta name="keywords" content="Isuroon, Health, Review, Doctors" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="author" content="Hack4Impact Carleton" />
+    <title>Isuroon MiniProj 1</title>
+    <link rel="stylesheet" href="../src/main.css" />
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+</head>
+
+<body>
+    <h1 id="Results"><i class="glyphicon glyphicon-chevron-left" id="back-button"></i> Results:</h1>
+
+    <div id="results_container">
+        <!-- Insert Doc Card here using CreateCards() script-->
```

Comment:
is this the same `createCards()` like one called on line 40? Is this comment still needed?